contracts/scripts/deployments/diamonds/DeploySpaceFactory.s.sol

```diff
@@ -212,6 +216,11 @@ contract DeploySpaceFactory is DiamondHelper, Deployer {
         legacyRuleEntitlement // legacyRuleEntitlement
       )
     );
+    addFacet(
```

Comment:
Well need to add this facet in https://github.com/river-build/river/blob/f7c2207602f41d7c5fb55055403722f4de3ccded/contracts/scripts/deployments/diamonds/DeploySpaceFactory.s.sol#L300 for alpha interaction ci script to upgrade this facet when it changes.

Reply:
yep, done in latest push